src/router.js

```diff
@@ -14,7 +14,7 @@ router.post('/register', (req, res, next) => {
 
 // login
 
-router.get('/login', (req, res, next) => {
+router.post('/login', (req, res, next) => {
   res.sendStatus(200);
   next();
 });
```

Comment:
for line 43 regarding route delete, again it is better to label the route by the product name so that it is easily referenced ie. router.delete('/clothes)